### DISCUSSION TITLE
S3 Gateway Support S3 Access point

### DISCUSSION THREAD

**Discussion by Sathyafish** [nginxinc/nginx-s3-gateway#82] [Q&A]
Will the Nginx S3 gateway support access objects using the S3 access point? I want to keep the traffic inside the VPC. My Nginx and S3 are in the same AWS account, So I don't need access to the S3 using a public endpoint.

**Comment by dekobon** [comment]
Hi @Sathyafish , does using the S3 gateway with a `S3_SERVER` set to the access point host not work? You can set up the gateway to talk to any S3 host and it doesn't need to be the public endpoint. I'm mostly ignorant of how the access points work, so more details on how the gateway isn't working with that use case would be appreciated.
